### ISSUE TITLE
Accidental upgrade

### ISSUE THREAD

**Issue by AnnoyedAsAlways** [OpenEmu/OpenEmu#2286]
I use OpenEmu on a Macbook Air 2012 running OS 10.7.5, and yesterday I decided to get the PSP emulation working. I downloaded the core, and while on that preference pane I decided to update all the cores I have. I then thought to check for a new update for OpenEmu and downloaded it, not realizing it was for OS 10.10 or higher. So I pulled my old version out of the trash and jettisoned the new one, but now hardly anything is working. 3DO, NES/Famicom Disk System, and Sega Saturn emulation are the only ones running. I'd like some advice on how to dig out, remove, and re-install the previous version cores. Or how to completely uninstall everything and start fresh (I'm not going to upgrade to OS 10.10, this Air has 4GB of non-upgradeable memory and Mavericks is a brutal memory pig on my Mac Mini with 8GB, so upgrading to El Capitan would make my Air pretty much unusable).

I'd appreciate any help you could give me.

**Comment by trumpetlicks** [comment]
I too am having some issues with this.  First of all the new version is compatible with 10.11 and higher, which I have to admit seems almost tooooooo forward reaching.  I have been looking through the code, and it really doesn't seem as if there are any REAL reasons this couldn't be compiled to work on 10.9 or 10.10.  I don't think they are using metal which would be the only real technological reason to use 10.11 forward.  What I think is NOT going on is, when the updater checks for updates, it really is only doing a simple check on the version only, without checking to see what the minimum OpenEmu version is necessary to use the core (or system for that matter).  There is a very manual way to fix this.  I have found that you can go into the OpenEmu repositories for the cores themselves and download the previous releases putting them in your ~Library/Application Support/OpenEmu/Cores folder.  You will obviously have to restart OpenEmu.  I am super surprised that the OpenEmu team would push so far forward so as to leave out any users of OSs < 10.11.  The reason that this bothers me so, is that OS 10.11 is horribly bad causing issues on many peoples' systems.  In my case it literally crashed my system to a point of unsuitability.  If one looks at the ratings for OS 10.11, they are less then 3 stars at this point, with MANY people downgrading back to 10.10 because of all the problems.  I really wish the OpenEmu team would work to recompile OpenEmu to work with 10.9 or 10.10 and higher.

**Comment by robotoad** [comment]
@AnnoyedAsAlways As mentioned, previous versions of the cores can be downloaded from their respective repositories.

@trumpetlicks Strongly disagree. I don't think app store ratings are an accurate barometer of quality in this case, because people not experiencing issues with their operating system are unlikely to go to the app store and set a rating. 10.9 compatibility is out of the question due to features like vibrancy and view controller containment, and while dependencies on 10.11 may be smaller in scope, larger changes that didn't make it into the first release require 10.11, such as the reimplementation of the grid view. For this and other reasons, we're committed to the decision.

**Comment by andijames** [comment]
I know this is slightly off the topic but FWIW my MacBook Air 2012 with 4GB RAM runs 10.11 supremely well. Memory management is much more efficient. Might be worth giving it a go.

**Comment by AnnoyedAsAlways** [comment]
I'm still wary of upgrading the Air because of how badly my desktop handled the upgrade. I can restart my Mini and without opening any programs I can watch the memory just bleed away. I upgraded to 8GB in the Mini and it still doesn't seem enough. Just using an external drive bleeds all my memory off within a minute or two. I've actually been traumatized by the memory handling after upgrading to 10.9, I don't want to cripple my Air the same way.

Thanks for the help trumpetlicks, I'll try and fix my mistakes later on tonight. Hopefully I can get everything running again. I've been grabbing screenshots of arcade games and pasting them onto their respective files so I don't have to look at hundreds of color bar codes in the Arcade section, I'd hate to have to start over with that.

**Comment by trumpetlicks** [comment]
@robotoad : I actually don't want to be argumentative, but I think the statement about people who aren't having problems not writing reviews is incorrect.  If you were correct, then how come previous releases had much higher ratings, and how come many apps have 4.5 star or higher ratings with thousands of votes.  The problems that I was seeing aren't just being talked about through the rating system but are being written about all over blog posts, etc...  I understand your choice from a feature perspective, but think that from a user perspective you might leave quite a few behind.  I guess that Im going to have to wait until OS 10.11 is stable enough to function on my system.

I should also say, that I LOVE THIS SYSTEM, which is the only reason I am upset with it pushing this far ahead.  It is rare to see a piece of software not at least support a version or two back.  Great work though, and I hope to be able to try out your new features at some point :-)

**Comment by robotoad** [comment]
@trumpetlicks  I don't dispute your experience, but people not having issues are less likely to write about it. I don't remember an OS X release that didn't receive complaints, even the ones looked back on fondly like Snow Leopard. It's unfortunate that anyone would feel cut off, but the fact that OS X is on an annual release schedule that hasn't increased system requirements in the last few updates was a contributing factor in the decision.

**Comment by trumpetlicks** [comment]
I would love to check out 2.0 :-)  A bit scared at this point due to my experience with El Capitan.  I have actually though about getting involved (as I am a software, electrical and systems engineer, and have also developed for OS X and iOS) with this project I like it so much.

**Comment by robotoad** [comment]
There's an IRC channel on Freenode if you'd like to follow development more directly.

**Comment by AnnoyedAsAlways** [comment]
I still can't get it to work. I trashed all the new cores and put the old ones in, and nothing is running, except one or two cores that were working before. I went into my system and eliminated everything that was OpenEmu related, including the preference files, and downloaded a fresh copy and ran that. It couldn't find any cores to run. So I double-clicked on the old cores that I had backed up on an external drive, and now it shows that each console has a core. But the games I dropped into it still don't play. I need to know what I'm missing so I can get rid of every trace of OpenEmu and start over. There's something I can't find that's keeping any system from working.
